packages/cli/src/export/__tests__/utils/createRelationship.ts

```diff
@@ -1,7 +1,9 @@
 import {
   Entity,
-  createDirectRelationship,
 } from '@jupiterone/integration-sdk-core';
+import {
+  createDirectRelationship
+} from '@jupiterone/integration-sdk-core/src';
```

Comment:
Not sure why the direct import from `@jupiterone/integration-sdk-core` stopped working for `createDirectRelationship` in this project. Whenever I import the `createDirectRelationship` without the ending `/src` on `@jupiterone/integration-sdk-core`  the function comes in undefined. I'm still trying to understand why this happens if anyone has any ideas.

Reply:
-_- Yup that was it, thanks Charlie.